### DISCUSSION TITLE
find_matches_to_new_records with Date types

### DISCUSSION THREAD

**Discussion by mb6852** [moj-analytical-services/splink#2461] [Q&A]
How do you find_matches_to_new_records where the model comparisons with dates are not as strings?

**Comment by ADBond** [comment]
This is a good question! I will check properly later, but I think probably the most reliable way would be to register your new records as a table with your backend, and then pass the name of this table to `find_matches_to_new_records()`.

Alternatively you might be able to pass records that have a `datetime.date` object. This will probably work better if you have a list of dicts (rather than a dict of lists), and might depend on the backend you use, but I _think_ it should work at least for `duckdb` and `spark`.
